.pre-commit-config.yaml

```diff
@@ -4,7 +4,7 @@ repos:
     hooks:
       - id: black
         language_version: python3
-  - repo: https://gitlab.com/pycqa/flake8
+  - repo: https://github.com/pycqa/flake8.git
```

Comment:
This should be already fixed by 7540e2e -- could you please rebase:
```
git remote add upstream https://github.com/popsim-consortium/stdpopsim.git
git fetch upstream
git checkout add_mus_mus
git rebase upstream/main
```
this will sync your branch to the main branch. To make sure everything looks OK, use `git log`. This should show your commits as the most recent, on top of 7540e2e.

Reply:
I think I've done this correctly but let's see I guess?

Regarding the different subspecies, I wouldn't necessarily say they have official references, I think people generally still map to GRCm38/39 (or some other such version related to the primary assembly). I can work on aggregating this information though. Certainly the vast amount of map/annotation/etc. information is going to be directed to GRCm3# for now.